### DISCUSSION TITLE
Do you want to allow your org to manage this device window popup?

### DISCUSSION THREAD

**Discussion by NateHutch365** [svrooij/WinTuner#33] [Q&A]
Curious to know, when publishing the app after the modern auth window I see the typical 'Do you want to allow your org to manage this device' window where I then uncheck the box and choose this app only - Is this supposed to happen and how would I clear that so I can sign into a different tenant?

**Comment by svrooij** [comment]
It should cache the accounts, and pick the one you specify with the Username parameter. That way you will only have to login once.

By using the Username and or Tenant parameter you can re-trigger the login. You cannot logout through the app, never considered that. It stores some encrypted file in the app data folder if you really want to remove the account cache.
